config/routes.rb

```diff
@@ -4,13 +4,18 @@
 
   namespace :api, path: nil, constraints: { subdomain: 'api' },
             defaults: { format: :json } do
-    resources :foods
-    resources :transactions
+    resources :foods do
+      member do
+        get 'quote'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.